ceremonies/src/tests.rs

```diff
@@ -1305,41 +1313,38 @@ fn grow_population_works() {
             .iter()
             .for_each(|p| register(account_id(&p), cid, None).unwrap());
 
-        let cindex = EncointerScheduler::current_ceremony_index();
         run_to_next_phase();
-        // ASSIGNING
-        assert_eq!(EncointerCeremonies::meetup_count((cid, cindex)), 1);
-        let meetup2_1 = EncointerCeremonies::meetup_registry((cid, cindex), 1);
 
+        // ASSIGNING
+        assert_eq!(get_meetups(cid).len(), 1);
         // whitepaper III-B Rule 3: no more than 1/3 participants without reputation
-        assert_eq!(meetup2_1.len(), 9);
+        assert_eq!(get_meetups_by_size(cid, 9).len(), 1);
 
         run_to_next_phase();
         // WITNESSING
-        fully_attest_meetup(cid, participants.clone(), 1);
+
+        fully_attest_meetups(cid, participants.clone());
 
         run_to_next_phase();
         // REGISTERING
 
-        let cindex = EncointerScheduler::current_ceremony_index();
         // register everybody again. also those who didn't have the chance last time
         for pair in participants.iter() {
-            let proof = get_proof(cid, cindex - 1, pair);
+            let proof = get_proof(cid, EncointerScheduler::current_ceremony_index() - 1, pair);
             register(account_id(&pair), cid, proof).unwrap();
         }
         run_to_next_phase();
+
         // ASSIGNING
-        assert_eq!(EncointerCeremonies::meetup_count((cid, cindex)), 2);
-        let meetup3_1 = EncointerCeremonies::meetup_registry((cid, cindex), 1);
-        let meetup3_2 = EncointerCeremonies::meetup_registry((cid, cindex), 2);
+        let meetups: Vec<Vec<AccountId>> = get_meetups(cid);
+        assert_eq!(meetups.len(), 2);
         // whitepaper III-B Rule 3: no more than 1/3 participants without reputation
-        assert_eq!(meetup3_1.len(), 7);
-        assert_eq!(meetup3_2.len(), 6);
+        assert_eq!(get_meetups_by_size(cid, 7).len(), 1);
```

Comment:
this a clever trick, but I would prefer something else.
What you actually want to test is the total number of participants assigned. So I'd rather collect all meetups, sum up the number of participants and we're good.

However, #68 will throw this all away so we shouldn't spend too much love for this. Leave as is plz

Reply:
could be easily implemented à la map(meetup -> meetup.len()).sum(), but I'll let it like this, also because this is actually a harder check than just the sum. It also verifies an equal distribution.